pyccel/ast/headers.py

```diff
@@ -89,11 +89,33 @@ def __getnewargs__(self):
         args = (self._args[0],)
         return args
 
+#==============================================================================
+class Template(Header):
+    """Represents a Template."""
```

Comment:
Could you please expand this docstring a bit? I think it needs a "Parameters" section, and an "Examples" section would also be very useful!